File: ConsoleAppVB1/Demo/DemoTemp.vb

Public Class DemoTemp
    Inherits DemoBase

    Shared Sub Run()
        Dim canExecute As Boolean = True
        Dim op As String = String.Empty
        Console.Clear()
        optionList.Clear()
        optionList.Add("Hello 001")                         '0

        While canExecute
            ShowOptions()
            op = Console.ReadLine()
            Console.WriteLine()
            Console.WriteLine("Your selection is： {0}", op)
            Console.WriteLine(LineString)
            Console.WriteLine()
            Select Case op
                Case "0"
                    SayHello()
                Case "exit"
                    canExecute = False
                Case Else
            End Select
        End While

        Console.WriteLine()

    End Sub
    Shared Sub SayHello()
        Console.Clear()
        Console.WriteLine(LineString)
        Console.WriteLine("hello.....")
    End Sub




End Class

File: ConsoleApp1/Demo/DemoTemp.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApp1.Demo
{

    class DemoTemp : DemoBase
    {

        public static void Run()
        {
            bool canExecute = true;
            string op = string.Empty;

            Console.Clear();
            optionList.Clear();
            optionList.Add("Hello...");                    //0

            while (canExecute)
            {
                ShowOptions();
                op = Console.ReadLine();
                Console.WriteLine();
                Console.WriteLine("Your selection is： {0}", op);
                Console.WriteLine();
                switch (op)
                {
                    case "0":
                        SayHello();
                        break;

                    case "exit":
                        canExecute = false;
                        break;
                    default:
                        break;
                }
            }
        }


        public static void SayHello()
        {
            Console.Clear();
            Console.WriteLine(Program.LineString);

            Console.WriteLine("......Hello World ....");
        }
    }
}
